cfn.yaml

```diff
@@ -0,0 +1,136 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: handle outbound messages from salesforce to update zuora and identity
+Parameters:
+  Stack:
+    Description: Stack name
+    Type: String
+    Default: membership
+  App:
+    Description: Application name
+    Type: String
+    Default: salesforce-message-handler
+  Stage:
+    Description: Stage name
+    Type: String
+    AllowedValues:
+      - CODE
+      - PROD
+    Default: CODE
+  DeployBucket:
+    Description: Bucket where RiffRaff uploads artifacts on deploy
+    Type: String
+    Default: membership-dist
```

Comment:
ah, should we be using the same bucket for all membership lambdas? I put my in a new dist bucket, but I wasn't sure if that was right